tce/opset/base/base.opp

```diff
@@ -4341,4 +4341,13 @@ Input operand is discarded.</description>
     <in element-count="1" element-width="1" id="1" type="RawData"/>
   </operation>
 
+  <operation>
+    <name>HWLOOP</name>
+    <description>Hardware-loop instruction to iterate given number of instruction. Operand-1: Iteration count and Operand-2: Number of instructions</description>
```

Comment:
Please describe also other (programmer) semantics relevant aspects. At least the fact that the loop doesn't have to fit the underlying cache construct and if it doesn't it still works for iteration counting.